client/cabal.project

```diff
@@ -0,0 +1,4 @@
+compiler: ghcjs
```

Comment:
In this config, the sub-projects (common, client, server) are quite independent. Regarding nix-shell, the workflow should be: 
- go in a sub-directory
- run a `nix-shell`
- write code and run `cabal build`
- exit the nix-shell and apply the same workflow in another sub-directory

That said, I wrote an alternative config (https://github.com/juliendehos/miso-isomorphic-example/tree/config2) with the following workflow:
- run `nix-shell -A client` in the main directory
- write code and run `cabal --builddir=dist-client --config-file=client.config build`
- exit + same with the similar server config

This may be simpler but I think the cabal commands are too complicated. This could be solved by adding an alias in the cabal2nix config (e.g. `alias cabal-server="cabal --builddir=dist-server --config-file=server.config"`) but I don't how to do that.